docs/07-faq/061-import-custom-terraform-modules.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Can I import custom Terraform Modules into Wing?
+id: import-custom-terraform-modules
+keywords: [faq, winglang, Wing programming language, Wing language, Terraform]
+---
+
+You can import custom Terraform modules into Wing if they have a CDKTF provider.
```

Comment:
Mmmm.... I don't think terraform modules have pre-built CDKTF providers. I think users will need to use `cdktf get` to generate the TypeScript code and publish it as a JSII library. It's a bit of a mess. We need to document this.

Reply:
Yes, they don't. This is why I wrote they can be used `if` they have a provider :) 
Can we say for now that we know it is not trivial to create the provider, but that if one was created it can be used?